src/Main.java

```diff
@@ -1,5 +1,103 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("\nTask1");
+//        С помощью цикла while посчитайте, сколько месяцев потребуется, чтобы накопить 2 459 000 рублей при условии,
+//        что первоначально мы имеем 0 рублей и готовы откладывать по 15 тысяч рублей.
+        int capital = 0, requiredAmount = 2459000, contribution = 15_000, i = 0;       //первоначально мы имеем 0 рублей
+        while (capital <= requiredAmount) {
+            i++;
+            capital = capital + contribution;
+            System.out.println("Месяц " + i + " , сумма накоплений равна " + capital + " рублей .");
+        }
+        System.out.println("Сумма накоплена за  " + i + " месяц, сумма накоплений равна " + capital + " рублей .");
+
+        System.out.println("\nTask2");
+        int number = 1;
+        while (number <= 10) {
+            System.out.print(number + " ");
+            number++;
+        }
+        System.out.println();
+
+        for (number = 10; number >= 1; number--) {
+            System.out.print(number + " ");
+
+        }
+        System.out.println("\nTask3");
+//     В стране Y население — 12 миллионов человек.
+//     Рождаемость составляет 17 человек на 1000, смертность — 8 человек.
+//     Рассчитайте, какая численность населения будет через 10 лет, если показатели рождаемости и смертности постоянны.
+        int population = 12_000_000, PopulationPer1000 = 0, birthPer1000 = 17, mortalityPer1000 = 8, year = 0;
```

Comment:
```suggestion
        int population = 12_000_000, bopulationPer1000 = 0, birthPer1000 = 17, mortalityPer1000 = 8, year = 0;
```
названия переменных должны быть с маленькой буквы всегда

Reply:
переменные надо объявлять по одной на каждой строке